### PR TITLE
Accessibility and Content changes

### PR DESCRIPTION
**Pull request by stephenjmcneill1971**
- Added "Open in new tab" for feedback link. 
- Added hint text for start page
- Removed unused span
- Changed text 'window' to 'tab'
- Changed Gov.uk destination to correct url
- Added address line text to all inputs address pages
- Changed hint test for 1st postcode in address pages
- Added aria-describedby attribute to address search input to link hint text
- Changed deceased address input labels for accessibility
- Removed redundant code

### CODE REVIEW

File: index/start-or-resume.njk [hunk 1]
```diff
@@ -37,7 +37,11 @@
                 },
                 {
                     "value": "resume",
-                    "text": "Continue a saved application"
+                    "text": "Continue a saved vvpplication",
```
Comment (tjbburton) [inline index/start-or-resume.njk:40]: Should be 'application' not 'vvpplication'

File: page/page.njk [hunk 2]
```diff
@@ -72,9 +72,9 @@
       tag: {
           text: "beta"
       },
-      html: 'This is a new service - your <a class="govuk-link" href="https://www.surveymonkey.co.uk/r/YourFeedbackPB" target="_blank">feedback<span class="govuk-visually-hidden">(opens in new window)</span></a> will help us to improve it.',
+      html: 'This is a new service - your <a class="govuk-link" href="https://www.surveymonkey.co.uk/r/YourFeedbackPB" target="_blank">feedback (opens in new window)<span class="govuk-visually-hidden"></span></a> will help us to improve it.',
```
Comment (tjbburton) [inline page/page.njk:75]: Do we still need the span tag now that we've taken the new window text out of it?

**Review by tjbburton — CHANGES_REQUESTED** [review]
Two small things to check

File: index/start-or-resume.njk (hunk 1 above)
Comment (stephenjmcneill1971) [reply index/start-or-resume.njk]: Changed

**Review by tjbburton — APPROVED** [review]
Looks good but test failed might need modifying to reflect changes

**Review by tjbburton — APPROVED** [review]

**Review by tjbburton — CHANGES_REQUESTED** [review]
I think you need to regenerate the bundle if you update the postcode-lookup text

**Review by tjbburton — APPROVED** [review]

File: questionnaire/questionnaireUISchema.js [hunk 3]
```diff
@@ -502,20 +496,14 @@ module.exports = {
...
                     }
                 },
                 'q-gp-building-and-street-3': {
                     options: {
                         macroOptions: {
-                            classes: '',
-                            label: {
-                                classes: 'govuk-visually-hidden'
-                            }
+                            classes: ''
```
[the remark this answers is not in the document]
Comment (stephenjmcneill1971) [reply questionnaire/questionnaireUISchema.js]: Removed

File: questionnaire/questionnaireUISchema.js [hunk 4]
```diff
@@ -556,20 +544,14 @@ module.exports = {
                 'q-applicant-treatment-building-and-street-2': {
                     options: {
                         macroOptions: {
-                            classes: '',
-                            label: {
-                                classes: 'govuk-visually-hidden'
-                            }
+                            classes: ''
```
[the remark this answers is not in the document]
Comment (stephenjmcneill1971) [reply questionnaire/questionnaireUISchema.js]: Removed

File: questionnaire/questionnaireUISchema.js [hunk 5]
```diff
@@ -556,20 +544,14 @@ module.exports = {
...
                     }
                 },
                 'q-applicant-treatment-building-and-street-3': {
                     options: {
                         macroOptions: {
-                            classes: '',
-                            label: {
-                                classes: 'govuk-visually-hidden'
-                            }
+                            classes: ''
```
[the remark this answers is not in the document]
Comment (stephenjmcneill1971) [reply questionnaire/questionnaireUISchema.js]: Removed

File: questionnaire/questionnaireUISchema.js [hunk 6]
```diff
@@ -1356,20 +1338,14 @@ module.exports = {
                 'q-applicant-dentist-address-building-and-street-2': {
                     options: {
                         macroOptions: {
-                            classes: '',
-                            label: {
-                                classes: 'govuk-visually-hidden'
-                            }
+                            classes: ''
```
[the remark this answers is not in the document]
Comment (stephenjmcneill1971) [reply questionnaire/questionnaireUISchema.js]: Removed

File: questionnaire/questionnaireUISchema.js [hunk 7]
```diff
@@ -1356,20 +1338,14 @@ module.exports = {
...
                     }
                 },
                 'q-applicant-dentist-address-building-and-street-3': {
                     options: {
                         macroOptions: {
-                            classes: '',
-                            label: {
-                                classes: 'govuk-visually-hidden'
-                            }
+                            classes: ''
```
[the remark this answers is not in the document]
Comment (stephenjmcneill1971) [reply questionnaire/questionnaireUISchema.js]: Removed

File: questionnaire/questionnaireUISchema.js [hunk 8]
```diff
@@ -1772,20 +1742,14 @@ module.exports = {
...
                     }
                 },
                 'q-rep-building-and-street-3': {
                     options: {
                         macroOptions: {
-                            classes: '',
-                            label: {
-                                classes: 'govuk-visually-hidden'
-                            }
+                            classes: ''
```
[the remark this answers is not in the document]
Comment (stephenjmcneill1971) [reply questionnaire/questionnaireUISchema.js]: Removed

File: questionnaire/questionnaireUISchema.js [hunk 9]
```diff
@@ -1820,20 +1784,14 @@ module.exports = {
...
```
[the remark this answers is not in the document]
Comment (stephenjmcneill1971) [reply questionnaire/questionnaireUISchema.js]: Removed

File: public/dist/js/bundle.js
Comment (neil-stephen-mcgonigle) [inline public/dist/js/bundle.js:1]: The bundle dosen't look correct? compared to master? 
It looks like it has been compiled with an earlier version of babel.

**Review by neil-stephen-mcgonigle — CHANGES_REQUESTED** [review]

**Review by neil-stephen-mcgonigle — APPROVED** [review]

**Pull request merged**